pkg/abap/aakaas/versionables.go

```diff
@@ -0,0 +1,144 @@
+package aakaas
+
+import (
```

Comment:
Looks nice as far as I can judge. My only comments are, as you can see, how many input error cases you want to catch...